### PR TITLE
Add support for package-lock.json file for GithubDependencyTreeTask

### PR DESCRIPTION
**Pull request by abs51295**
- Currently, mercator only supports npm-shrinkwrap.json and hence we need a hack to support package-lock.json as of now.
- Need to revert this once mercator supports package-lock.json.

Signed-off-by: abs51295 <aagams68@gmail.com>

### CODE REVIEW

**Comment by abs51295** [comment]
Working on adding tests :)

**Comment by abs51295** [comment]
Please merge #615 first :)

**Comment by msrb** [comment]
[test]

File: f8a_worker/workers/dependency_parser.py [hunk 1]
```diff
@@ -144,3 +148,17 @@ def get_npm_dependencies(cls, path):
...
+    @staticmethod
+    def change_package_lock_to_shrinkwrap():
+        """Rename package-lock.json to npm-shrinkwrap.json.
+
+        For more information about package-lock.json please visit
+        https://docs.npmjs.com/files/package-lock.json
+        """
+        # TODO: Remove this method once mercator has support for package-lock.json
+
+        package_lock_path = Path.cwd() / "package-lock.json"
```
Comment (msrb) [inline f8a_worker/workers/dependency_parser.py]: Does this reliably work? I mean, can we expect CWD to be correct when we hit this line?

**Comment by msrb** [comment]
@abs51295 just a question, otherwise LGTM.

File: f8a_worker/workers/dependency_parser.py (hunk 1 above)
Comment (abs51295) [reply f8a_worker/workers/dependency_parser.py]: Yes, because we are in correct directory in line 43 itself.

Comment (msrb) [reply f8a_worker/workers/dependency_parser.py]: I see it now, thanks :wink:

**Pull request merged**